Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheet.swift

```diff
@@ -313,7 +313,7 @@ class PhotonActionSheet: UIViewController, UITableViewDelegate, UITableViewDataS
     func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
         if section == 0 {
             if site != nil {
-                return PhotonActionSheetUX.TitleHeaderSectionHeightWithSite
+                return 130
```

Comment:
I wonder if it would make sense to create a static variable for this value :thinking: